test/parser/mixins.spec.js

```diff
@@ -25,6 +25,14 @@ describe('Parser', () => {
             expect(root.first.type).to.eql('atrule');
         });
 
+        it('should parse parametric mixins', () => {
+            const root = parse('.foo(@bar) { color: @bar;}', {mixinsAsAtRules: true, innerMixinsAsRules: true});
+
+            expect(root.first.type).to.eql('atrule');
+            expect(root.first.params[0].type).to.eql('atrule');
```

Comment:
@shellscape, I see your point. I think you're right, I have to extend the parsed tree from `postcss-less` and use not only   nodes from API of PostCSS, but all custom nodes should have similar syntax and structure as nodes AtRule, Rule, etc. I think it will be done in the next version of postcss-less